project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -690,3 +713,23 @@ function c4m_user_og_mail_tokens(array &$replacements, array $data, $options) {
   c4m_content_group_mail_tokens($replacements, $data, $options);
   user_registrationpassword_mail_tokens($replacements, $data, $options);
 }
+
+/**
+ * Implements hook_user_update().
+ */
+function c4m_user_og_user_update(&$edit, $account, $category) {
+  if ($account->original->status || !$account->status) {
+    return;
+  }
+
+  // Associate the user to the group.
+  foreach (c4m_user_og_fetch_unified_workflow_entries($account->uid) as $entry) {
+    $group = node_load($entry->gid);
+    $membership_request_value = c4m_og_get_group_membership_request($group);
+    $values = array(
+      'user' => user_load($entry->uid),
+      'state' => $membership_request_value == 'open' ? OG_STATE_ACTIVE : OG_STATE_PENDING ,
+    );
+    og_group('node', $entry->gid, $values);
```

Comment:
@amitaibu 
I get this error over here:
![selection_230](https://cloud.githubusercontent.com/assets/5985683/16920909/2b76479c-4d18-11e6-81e2-e9d77a21d2ec.png)

any suggestions ?

thanks ahead :)